app/webpacker/scripts/global/disable-browser-autofill.js

```diff
@@ -8,10 +23,27 @@ function disableBrowserAutofill () {
   }
 
   document.querySelectorAll('form').forEach(form => {
-    if (form.getAttribute('autocomplete') === 'off') {
-      const inputs = [...form.querySelectorAll('input[autocomplete]')].filter(input => inputTypes.includes(input.type))
+
+    let inputs = []
+
+    if (form.dataset.jsDisableBrowserAutofill === 'on'){
+      inputs = [...form.querySelectorAll('input')]
+      .filter(input => inputTypes.includes(input.type))
+      .filter(input => input.dataset.jsDisableBrowserAutofill !== 'off')
+    }
+
+    else {
```

Comment:
`Else`where, we're writing if/else statements like:

```
if (...) {
  ...
} else {
  ...
}
```

Might be good to change for consistency, but also just a personal preference thing.